src/styles/_shame.scss

```diff
@@ -0,0 +1,160 @@
+.container {
+  margin: 0 auto;
+  max-width: 600px;
+  padding: 0 1rem;
+}
+
+.container--main {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  min-height: calc(100vh - 5.75rem);
+}
+
+.content {
+  font-size: 1.125rem;
+  font-weight: 300;
+
+  &__image {
+    display: block;
+    margin: 0 auto 1rem;
+    max-width: 192px;
+    text-align: center;
+
+    line {
+      transition: transform 0.2s ease-in-out;
+    }
+
+    &:hover line {
+      transform: translateY(28%);
+    }
+  }
+}
+
+.tests {
+  list-style: none;
+  margin: 0 auto 1rem;
+  padding-left: 0;
+  width: 15ch;
+}
+
+.test {
+  animation: reveal 0.3s forwards cubic-bezier(0.5, -0.5, 0.5, 1.5);
+  display: block;
+  opacity: 0;
+  transform: scale(0) translateX(-1rem);
+  transition: 0.5s color ease-in-out;
+
+  @for $i from 1 through 3 {
+    &:nth-of-type(#{$i}) {
+      animation-delay: 0.2s * $i;
+    }
+
+    &::before {
+      display: inline-block;
+      margin-right: 1ch;
+      transition: 0.3s transform ease-in-out;
+    }
+    &--correct {
+      color: color(teal, dark);
+
+      &::before {
+        content: '👍';
+      }
+    }
+
+    &--incorrect {
+      color: color(red, dark);
+      text-decoration: line-through;
+
+      &::before {
+        content: '💩';
+      }
+    }
```

Comment:
This block should be outside of the for loop. The for loop only contains the `nth-of-type` selector in the original: 

https://github.com/codesect/gulp-starter/blob/master/src/styles/_shame.sass#L42-L44

Reply:
Oh right, my bad :)